src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -148,11 +148,11 @@ public override void Initialize(ViewWithData data)
 
             pullRequests.OriginalCompleted
                 .ObserveOn(RxApp.MainThreadScheduler)
-                .Catch<System.Reactive.Unit, Octokit.AuthorizationException>(ex =>
-                {
-                    log.Info("Received AuthorizationException reading pull requests", ex);
-                    return repositoryHost.LogOut();
-                })
+                ////.Catch<System.Reactive.Unit, Octokit.AuthorizationException>(ex =>
+                ////{
+                ////    log.Info("Received AuthorizationException reading pull requests", ex);
+                ////    return repositoryHost.LogOut();
+                ////})
```

Comment:
I think this code should be removed @shana - as far as I can see there's no good reason for the PR list to be messing with removing connections?